datafusion/common/src/scalar.rs

```diff
@@ -3810,4 +3918,4 @@ mod tests {
             ]
         );
     }
-}
+}
```

Comment:
**nitpick:** missing EOL at the end of the file

Reply:
Thanks for the hint! I didn't really know about this requirement.